### ISSUE TITLE
Pipeline:report keeps crashing

### ISSUE THREAD

**Issue by sgraindorge** [epi2me-labs/wf-clone-validation#28]
### Operating System

macOS

### Workflow Version

v0.4.0-g344e1d4

### Workflow Execution

Command line

### CLI command run

nextflow run epi2me-labs/wf-clone-validation \ 
	--fastq '/Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/20230711_1544_MN38795_AIY998_aaa6be64/fastq_pass' \ 
	--assm_coverage 2 \
	--basecalle_cfg dna_r9.4.1_e8_fast@v3.4 \
	--sample_sheet '/Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/20230711_1544_MN38795_AIY998_aaa6be64/sample_sheet.csv'

### Workflow Execution - CLI Execution Profile

None

### What happened?

The pipeline runs well until the report.
There is an error message I don't understand. So any help / clue would be great !

### Relevant log output

```shell
Nextflow workflow report
[clever_neumann]
Workflow execution completed unsuccessfully!
The exit status of the task that caused the workflow execution to fail was: 1.

The full error message was:

Error executing process > 'pipeline:report (1)'

Caused by:
  Process `pipeline:report (1)` terminated with an error exit status (1)

Command executed:

  workflow-glue report     --downsampled_stats downsampled_stats/*     --revision master     --commit 344e1d419a4af855624b64563fe8a7c738f0b962     --status final_status.csv     --per_barcode_stats per_barcode_stats/*     --host_filter_stats host_filter_stats/*     --params params.json     --versions versions     --report_name wf-clone-validation-report.html     --plannotate_json plannotate_report.json     --lengths plannotate.json     --inserts_json insert_data.json     --qc_inserts qc_inserts/*     --assembly_quality assembly_quality/*

Command exit status:
  1

Command output:
  (empty)

Command error:
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:107: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:119: DeprecationWarning: textsize is deprecated and will be removed in Pillow 10 (2023-07-01). Use textbbox or textlength instead.
    width, height = draw.textsize(self.css_icons[icon], font=font)
  /home/epi2melabs/conda/lib/python3.8/site-packages/icon_font_to_png/icon_font.py:164: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
    out_image = out_image.resize((org_size, org_size), Image.ANTIALIAS)
  Traceback (most recent call last):
    File "/Users/sgraindorge/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/workflow-glue", line 7, in 
      cli()
    File "/Users/sgraindorge/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/workflow_glue/__init__.py", line 72, in cli
      args.func(args)
    File "/Users/sgraindorge/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/workflow_glue/report.py", line 279, in main
      qual_df = read_files(
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/core/frame.py", line 3464, in __getitem__
      indexer = self.loc._get_listlike_indexer(key, axis=1)[1]
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/core/indexing.py", line 1314, in _get_listlike_indexer
      self._validate_read_indexer(keyarr, indexer, axis)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/core/indexing.py", line 1374, in _validate_read_indexer
      raise KeyError(f"None of [{key}] are in the [{axis_name}]")
  KeyError: "None of [Index(['sample_name', 'mean_quality'], dtype='object')] are in the [columns]"

Work dir:
  /Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/sequanix/work/38/d8fe283410e67c2ef4616ec23a7d71

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`
Run times
17-Aug-2023 10:52:08 - 17-Aug-2023 11:04:02 (duration: 11m 54s)
  55 succeeded  
  49 failed  
Nextflow command
nextflow run epi2me-labs/wf-clone-validation --fastq /Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/20230711_1544_MN38795_AIY998_aaa6be64/fastq_pass --assm_coverage 2 --basecaller_cfg dna_r9.4.1_e8_fast@v3.4 --sample_sheet /Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/20230711_1544_MN38795_AIY998_aaa6be64/sample_sheet.csv
CPU-Hours
0.7 (80,9% failed)
Launch directory
/Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/sequanix
Work directory
/Volumes/BioSan/Users/sgraindorge/AEG_Assemblage-plasmides-nanopore/sequanix/work
Project directory
/Users/sgraindorge/.nextflow/assets/epi2me-labs/wf-clone-validation
Script name
main.nf
Script ID
7a2123c22dfa1374e4c724c58dae4e72
Workflow session
64227bfc-08d6-46ea-a2ac-9d72d1f11398
Workflow repository
https://github.com/epi2me-labs/wf-clone-validation, revision master (commit hash 344e1d419a4af855624b64563fe8a7c738f0b962)
Workflow profile
standard
Nextflow version
version 23.04.2, build 5870 (08-06-2023 08:29 UTC)
```

**Comment by sarahjeeeze** [comment]
Hi, I also found this bug in v0.4.0. It should be fixed in v0.5.0.

**Comment by sarahjeeeze** [comment]
Closing as this is now fixed.
